packages/Config/RectorConfig.php

```diff
@@ -223,7 +223,20 @@ static function (
 
     public function import(string $filePath): void
     {
-        $this->importFile($filePath);
+        if (str_contains($filePath, '*')) {
+            throw new ShouldNotHappenException(
+                'Matching file paths by using glob($filePath) is no longer supported. Use specific file path instead.'
+            );
+        }
```

Comment:
@TomasVotruba @staabm @rmobis this is ensure user see the reason why `*` using glob on import no longer supported.

Reply:
I had the same thing in mind - great!

Reply:
@staabm do you have better message to throw?

Reply:
That's awesome, thanks.